### PR TITLE
Fixing issue #5550 where empty accessory files makes manual handling erratic

### PR DESCRIPTION
**Pull request by zonkmachine**
Fixes https://github.com/pioneerspacesim/pioneer/issues/5550

Here is a fix for #5550.
The Character faces now have an accessory generated about one time in four. Player character, bbs faces or random button in the face app in the 'Personal Information' window all works. When you step the accessories in the face app they will show up all about the 1st step which is an empty file that was kept so you can intentionally select 'no accessory'.

### CODE REVIEW

**Review by Web-eWorks — APPROVED** [review]

File: data/pigui/libs/face.lua [hunk 1]
```diff
@@ -36,7 +36,7 @@ local function rerollFaceDesc(oldDesc)
 		FEATURE_NOSE = Engine.rand:Integer() % 2^31,
 		FEATURE_MOUTH = Engine.rand:Integer() % 2^31,
 		FEATURE_HAIRSTYLE = Engine.rand:Integer() % 2^31,
-		FEATURE_ACCESSORIES = Engine.rand:Integer() % 2^31,
+		FEATURE_ACCESSORIES = Engine.rand:Integer() % 4 == 0 and Engine.rand:Integer() % 2^31 or 0, -- accessory on one out of four faces
```
Comment (Gliese852) [inline data/pigui/libs/face.lua:39]: I guess the `and` branch can also generate an empty accessory?

Comment (zonkmachine) [reply data/pigui/libs/face.lua:39]: Correct. With the current number of accessories every eight png will be invisible.

**Comment by impaktor** [comment]
Did we used to have santa hat for our faces, when computer clock is Christmas, or was it Frontier, or did I dream that?
Either way, would be fun easter egg, when playing during Christmas break.

**Comment by zonkmachine** [comment]
> Did we used to have santa hat for our faces, when computer clock is Christmas, or was it Frontier, or did I dream that?

I just experimented with a santa hat and also a hijab to test the layers of facegen. Are any of those actually in this PR?! I can't find them in here.

**Comment by Web-eWorks** [comment]
Looks good, thanks @zonkmachine!

**Pull request merged**